astropy/wcs/tests/test_wcs.py

```diff
@@ -676,29 +686,31 @@ def test_out_of_bounds():
 
 def test_calc_footprint_1():
     fits = get_pkg_data_filename('data/sip.fits')
-    w = wcs.WCS(fits)
-
-    axes = (1000, 1051)
-    ref = np.array([[202.39314493, 47.17753352],
-                    [202.71885939, 46.94630488],
-                    [202.94631893, 47.15855022],
-                    [202.72053428, 47.37893142]])
-    footprint = w.calc_footprint(axes=axes)
-    assert_allclose(footprint, ref)
+    with pytest.raises(FITSFixedWarning):
```

Comment:
For some of these tests, I need to indent the whole block or Python complains about undefined variable (that was defined inside the block). :woman_shrugging: